build/foss/Cargo.toml

```diff
@@ -8,5 +8,7 @@ rust-version = "1.72.1"
 crate-type = ["staticlib"]
 
 [dependencies]
-norddrop = { git = "https://github.com/NordSecurity/libdrop", tag = "v5.4.0_moose_backport" }
-telio = { git = "https://github.com/NordSecurity/libtelio", tag= "v4.3.3" }
+# norddrop = { git = "https://github.com/NordSecurity/libdrop", tag = "v7.0.0" }
+# FIXME: Change it
+norddrop = { path = "../../../libdrop/norddrop" }
```

Comment:
FYI: It's just to make the integration easier because libdrop moved entirely to `cdylib`, so I added a `lib` to `crate-type` locally to postpone dynamic linking until I have everything working.